samples/ControlCatalog.NetCore/Program.cs

```diff
@@ -111,10 +113,20 @@ public static AppBuilder BuildAvaloniaApp()
                     EnableMultiTouch = true,
                     UseDBusMenu = true,
                     EnableIme = true,
+                    UseGpu = true,
+                    UseVulkan = true
                 })
                 .With(new Win32PlatformOptions
                 {
-                    EnableMultitouch = true
+                    EnableMultitouch = true,
+                    UseWgl = true,
+                    UseVulkan = true,
+                })
+                .With(new VulkanOptions()
+                {
+                    UseDebug = true,
+                    PreferDiscreteGpu = true,
+                    PreferredDevice = 8085
```

Comment:
Should be removed since I guess this is your device id.

Reply:
Yup, forgot that was there.